packages/ui/src/working-groups/hooks/useApplications.ts

```diff
@@ -18,10 +22,14 @@ export const ApplicationStatusToTypename: Record<ApplicationStatus, ApplicationS
   withdrawn: 'ApplicationStatusWithdrawn',
 }
 
-export const useApplications = ({ applicationsStatus, openingId }: Props) => {
+export const useApplications = ({ applicationsStatus, openingId, stakingAccount, limit, skip, orderBy }: Props) => {
   const { loading, data } = useGetWorkingGroupApplicationsQuery({
+    skip,
     variables: {
+      limit,
+      orderBy,
```

Comment:
Here I don't think `orderBy` has to be a parameter, it's just nicer to make it explicitly `CreatedAtDesc` (IMO it makes just as much sense to order it this way in `SelectWorkingGroupApplicationBase`)
```suggestion
      orderBy: [WorkingGroupApplicationOrderByInput.CreatedAtDesc],
```